google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -168,6 +168,16 @@ void getAll(
   @Nonnull
   WriteBatch batch();
 
+  /**
+   * Returns a FirestoreBundle.Builder {@link FirestoreBundle.Builder} instance for the given bundle
+   * Id.
+   *
+   * @param bundleId If null, an random string will be used for the Id. The Id is used by the client
+   *     SDKs loading bundles to identify if a bundle has been loaded before.
```

Comment:
s/Id/ID and swap both sentences.

If a null bundle ID is how me mainly expect the bundles to be used, should we add an overload that doesn't require an explicit `null`?

Reply:
Yep, that is better.

I also realized we provided no way for the client to get the bundle id if it is auto-id, they probably want to use it to manage their bundle serving. I added a `getId` to builder. Will pass a note to the council again.

Reply:
Thought on adding `bundleBuilder()` with no arguments? I believe this would be the main way to interact with bundles, and we should make it as easy as possible.